packages/shared/src/libs/helpers/format-date/format-date.helper.ts

```diff
@@ -0,0 +1,7 @@
+import { format } from "date-fns";
+
+const formatDate = (createdAt: string): string => {
```

Comment:
If this is a common helper it should accept any date and allowed formats
```suggestion
type DateFormat = 'd MMM yyyy HH:mm'

const formatDate = (date: Date, format: DateFormat): string => {
```